src/config/config.ts

```diff
@@ -1,6 +1,10 @@
 const environmentUrls = new Map();
 
 environmentUrls.set("localhost", "http://localhost:8080");
+environmentUrls.set(
+    "compstore-ui.onrender.com",
+    "https://compstore-api.onrender.com",
+);
```

Comment:
To ustawienie służy do ustalenia na jaki adres mają być wysyłane żądania do backendu. `window.location.hostname` sprawdza hosta frontu, jak znajdzie:
- localhost -> to będzie odpytywać http://localhost:8080
- compstore-ui.onrender.com -> https://compstore-api.onrender.com

Aby to działo trzeba w Cross-Origin na backend dodać `https://compstore-ui.onrender.com` do pliku https://github.com/grz55/compstore-api/blob/dev/src/main/java/com/compstore/config/CrossOriginConfig.java

Reply:
Ok, w otwartym branchu na backend dodałem odnośnik do podanego frontendu, który będzie uzupełniany tak samo jak dane połączenia z BD, czyli przez zmienne VM